featuretools/tests/entityset_tests/test_ww_es.py

```diff
@@ -624,23 +679,62 @@ def test_update_dataframe_different_dataframe_types():
         dask_es.update_dataframe('sessions', sessions)
 
 
-def test_update_dataframe_last_time_index(es):
-    # --> koalas currently fails bc we can't get the schema because it deepcopies
+def test_update_dataframe_and_min_last_time_index(es):
+    es.add_last_time_indexes(['products'])
+
+    original_time_index = es['log']['datetime'].copy()
+    original_last_time_index = es['products']['last_time'].copy()
+
+    if ks and isinstance(original_time_index, ks.Series):
+        new_time_index = ks.from_pandas(original_time_index.to_pandas() + pd.Timedelta(days=1))
+        expected_last_time_index = ks.from_pandas(original_last_time_index.to_pandas() + pd.Timedelta(days=1))
+    else:
+        new_time_index = original_time_index + pd.Timedelta(days=1)
+        expected_last_time_index = original_last_time_index + pd.Timedelta(days=1)
+
+    new_dataframe = es['log'].copy()
+    new_dataframe['datetime'] = new_time_index
+
+    es.update_dataframe('log', new_dataframe, recalculate_last_time_indexes=True)
```

Comment:
This might also be an issue for the tests that follow this one.

Reply:
Oof, yeah, this definitely is something we need to handle. Good catch--added. 

In the case where we aren't recalculating last time index and the new dataframe has no last time index column, I opted to lose any reference to a last time index on the DataFrame, since the new dataframe won't have a last time index column added.

Reply:
> In the case where we aren't recalculating last time index and the new dataframe has no last time index column, I opted to lose any reference to a last time index on the DataFrame, since the new dataframe won't have a last time index column added.

Seems reasonable